documentation/slate/source/examples/jsFiddle/journeys/demo.css

```diff
@@ -0,0 +1,64 @@
+body {
+  font-family: Helvetica, Sans, Arial;
+}
+
+p:first-child {
+  margin: 20px;
+}
+
+table {
+    font-size: 16px;
```

Comment:
CSS in px ? Em and ex are better units especially these days with retina qhd too much pixels...